auth/views.py

```diff
@@ -25,7 +25,7 @@ def login():
             errors.append(invalid)
         else:
             errors.append(invalid)
-    return render_template('auth/login.html', errors=errors)
+    return render_template('auth/login.html', errors=errors), 401
```

Comment:
This is a very good point!
However you are returning an error 401 in every case here…
Maybe it would be interesting to cover this case with you tests, and then, once you can see that it fails (status code not being 200) when you just land on the page, you could adapt here ;)